lib/get/onValue.js

```diff
@@ -34,6 +34,11 @@ module.exports = function onValue(model, node, seed, depth, outerResults,
         valueNode = clone(node);
     }
 
+    // We don't want to emit references in json output
+    else if (!isJSONG && node.$type === $ref) {
+        valueNode = undefined;
```

Comment:
Do we want to emit an empty branch node here instead? Though perhaps not until we start emitting branch nodes we know to be present.